src/content/h1/tools/mek/readme.md

```diff
@@ -1,17 +1,21 @@
 ---
 title: Mozz Editing Kit (MEK)
-template: tool
+toolName: MEK
 img: screenshot.png
 imgCaption: Screenshot of the MEK tool Mozzarilla, a replacement for [Guerilla][]
 info: |
-  [Development thread](https://opencarnage.net/index.php?/topic/7877-mozz-editing-kit-mek/)
+  * [Source](https://github.com/Sigmmma)
+  * [Development thread](https://opencarnage.net/index.php?/topic/7877-mozz-editing-kit-mek/)
```

Comment:
The threads never really get updated. Might consider putting something else in its place here

Reply:
I'll put our discord as a support channel